src/composeWithStateSync.ts

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable @typescript-eslint/ban-types */
+
+import { StoreEnhancer, StoreEnhancerStoreCreator } from 'redux'
+import { forwardAction } from './forwardAction'
+import { MainStateSyncEnhancerOptions } from './options/MainStateSyncEnhancerOptions'
+import { RendererStateSyncEnhancerOptions } from './options/RendererStateSyncEnhancerOptions'
+import { stateSyncEnhancer } from './stateSyncEnhancer'
+
+export type StateSyncOptions = MainStateSyncEnhancerOptions | RendererStateSyncEnhancerOptions
```

Comment:
This looks like a circular type dependency to me => `MainStateSyncEnhancerOptions` extends the `StateSyncOptions` in the other place, yet here it is an union?

Reply:
Good find, this was "legacy" and can be removed.